ServerAutomation/api/makeACall.js

```diff
@@ -12,7 +12,7 @@ function createCall(number, ncco) {
   nexmo.calls.create(
   {
     to: [{ type: 'phone', number: number }],
-    from: { type: 'phone', number: '917276393663' },
+    from: { type: 'phone', number: '919999999999' },
```

Comment:
Actually I would prefer to keep it as 91XXXXXXXXXX in github, and change accordingly in use/production. I missed it while commit. So, you can do this in this commit.

Reply:
Updated the change

Reply:
could you please squash both the commits to make a single commit as they account for single change. And make some meaningful commit message as it helps to read the change from looking atop.

Reply:
@reasonDheeraj could you please